GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1962,6 +1962,13 @@ public class ChangeCount
             public IReadOnlyList<GitItemStatus> Deleted { get; set; }
             public IReadOnlyList<GitItemStatus> SubmodulesChanged { get; set; }
             public IReadOnlyList<GitItemStatus> SubmodulesDirty { get; set; }
+
+            public bool IsEmpty
```

Comment:
I would have preferred HasChanges considering how this is used, but I will not request changes

Reply:
Renaming to HasChanges should also invert the logic...

Reply:
> Renaming to HasChanges should also invert the logic...

:man_facepalming:
renamed to `Any`